Formula/osh.rb

```diff
@@ -1,8 +1,8 @@
 class Osh < Formula
   desc "Two ports of /bin/sh from V6 UNIX (circa 1975)"
   homepage "http://v6shell.org"
-  url "http://v6shell.org/src/osh-20160108.tar.gz"
-  sha256 "51980473429cd1f8dad255c30749b8588278caf42b28185e22b9455e8f381bf7"
+  url "http://v6shell.org/src/osh-4.2.1.tar.gz"
```

Comment:
I wonder if we can devise an `audit` check that at least complains that a PR downgrades a formula, so this won't be flagged as ✅ and won't be merged without scrutiny. (Could probably be part of the existing `audit` check that walks Git history and makes sure that formula `revision`s are properly incremented and reset on version bump.)